### ISSUE TITLE
Top level graph input config

### ISSUE THREAD

**Issue by amarrella** [dagster-io/dagster#5828]
## Summary

hi :wave: i saw that #5732 added the possibility of configure graph/job input, but this doesn't seem to be reflected in the dagit playground (i cannot set graph input from there). Is it something that will be added? 
It's not possible to set the input at the graph level nor the op level.

## Reproduction
```python
import dagster

@dagster.op
def my_op(x: str) -> str:
    return x

@dagster.graph
def my_graph(x: str) -> str:
    return my_op(x=x)

my_job = my_graph.to_job()

@dagster.repository
def my_repo():
    return [my_job]
```

## Dagit UI/UX Issue Screenshots

<img width="1336" alt="Screen Shot 2021-11-30 at 14 27 07" src="https://user-images.githubusercontent.com/1425877/144055739-21c044bc-2f26-4e5c-aaa4-009ff1f3b8d9.png">
<img width="1329" alt="Screen Shot 2021-11-30 at 14 26 55" src="https://user-images.githubusercontent.com/1425877/144055743-66ff7528-d154-481b-a673-6450a7bf5960.png">
<img width="883" alt="Screen Shot 2021-11-30 at 14 26 47" src="https://user-images.githubusercontent.com/1425877/144055747-cbfaa187-3c35-4553-a99c-349a3159e40e.png">
<img width="439" alt="Screen Shot 2021-11-30 at 14 27 42" src="https://user-images.githubusercontent.com/1425877/144055822-4a6880a0-c03c-4535-8b45-95b866d3d6d1.png">

## Additional Info about Your Environment

dagster 0.13.9

---
#### Message from the maintainers:
Impacted by this bug? Give it a 👍. We factor engagement into prioritization.

**Comment by yuhan** [comment]
Hi @amarrella thanks for reporting and trying the new change! The change was not released yet with 0.13.9. It will get out this Thur with 0.13.10.

**Comment by dpeng817** [comment]
Confirmed that the posted snippet works on master. Sorry for the inconvenience!
